### ISSUE TITLE
Handle checkable list view controls as list views with columns

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#8857]
### Steps to reproduce:
1. Install the "Add-on Updater" add-on, since as far as I know, for the time being that is the only component for NVDA that has the new checkable list view controls with multiple columns.
2. Make sure that you have at least 2 add-ons that require updating.
3. Check for updates for your installed add-ons, by going to NVDA menu -> Tools -> Check for add-on updates.
4. When the dialog with the available add-on updates appears, focus the checkable list view control listing the available add-on updates.
5. Use NVDA's table navigation commands (Control+Alt+Arrows by default) to navigate between rows and columns in the checkable list view control.

### Actual behavior:
NVDA's table navigation commands appear not to work for this type of checkable list view control, as they do for standard list view controls with multiple columns. NVDA stays silent when the commands are executed.

### Expected behavior:
NVDA's table navigation commands should work for this type of checkable list view control, as they do for standard list view controls with multiple columns. NVDA should be able to read information column by column in this type of checkable list view control, as it does in standard list view controls.

### System configuration:
- Operating system: Windows 10 Pro version 1809 (build 17763.55), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: alpha-16164,8a7da1d4.
- "Add-on Updater" add-on version: 20181018.
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

#### Does the issue still occur after restarting your PC?
Yes, it does.

#### Have you tried any other versions of NVDA?
No.

**Comment by leonardder** [comment]
This is probably a one liner to accomplish and could best be combined with #8554. However, it will change the look and feel for these lists a bit. I.e. instead of
- Downloaded Program Files check box checked 0 bytes 1 of 10
It would be
- Downloaded Program Files; 0 bytes check box checked 1 of 10

This could get more complex for lists with more columns. I'd be an advocate of this change, though.
